doc/interop-test-descriptions.md

```diff
@@ -85,56 +83,96 @@ Procedure:
 
 Client asserts:
 * call was successful
-* response payload type is COMPRESSABLE
 * response payload body is 314159 bytes in size
 * clients are free to assert that the response payload body contents are zero
   and comparing the entire response message against a golden response
 
-### large_compressed_unary
+### client_compressed_unary
 
-This test verifies compressed unary calls succeed in sending messages. It
-sends one unary request for every payload type, with and without requesting a
-compressed response from the server.
+This test verifies the client can compress unary messages. It sends an initial
+inconsistent request to verify whether the server supports the
+[CompressedRequest][] feature. If it does, it should catch the inconsistency and
```

Comment:
I'd say it's better to just describe the idea of the test itself first (send one compressed and on uncompressed call) and then perhaps mention the details about sending a probe request. This description starts with the less important thing and only mentions the most important things later.

"If it does"  should be If it doesn't.